tests/unit/test_credentials.py

```diff
@@ -54,10 +54,19 @@ def path(filename):
 
 
 class TestCredentials(BaseEnvVar):
-    def test_detect_nonascii_character(self):
-        c = credentials.Credentials('foo\xe2\x80\x99', 'bar\xe2\x80\x99')
+    def _ensure_credential_is_normalized_as_unicode(self, access, secret):
+        c = credentials.Credentials(access, secret)
         self.assertTrue(isinstance(c.access_key, type(u'u')))
         self.assertTrue(isinstance(c.secret_key, type(u'u')))
+        return True
+
+    def test_detect_nonascii_character(self):
+        self.assertTrue(self._ensure_credential_is_normalized_as_unicode(
```

Comment:
I'm also ok with not having the `assertTrue` call and just calling the _ensure method in this test..  The `_ensure_credential_is_normalized_as_unicode` will raise an AssertionError if it fails anways so it simplifies the test a bit.